pak_wheels.py

```diff
@@ -1,186 +0,0 @@
-        return dictionary[attrib]
-    except KeyError:
-        pass
-    return None
-
-
-def get_array_attribute_safely(dictionary, main_attrib, sub_attrib):
-    """Returns None if attribute not found in dictionary
-
-        Arguments:
-            dictionary: dict:
-                Dictionary that has values of the attributes
-            main_attrib: string:
-                Parent attribute to refer to child
-            sub_attrib: string:
-                Child attribute whose value is required
-
-        Returns:
-             Value of child attribute if exists, otherwise returns None
-    """
-
-    try:
-        return dictionary[main_attrib][sub_attrib]
-    except KeyError:
-        pass
-    return None
-
-
-def get_clean_dictionary(dictionary):
-    """Creates a clean dictionary with useful attribute
-
-    Arguments:
-        dictionary: dict:
-            Dictionary from where arguments will be extracted
-
-    Returns:
-        dictionary: dict:
-            Which contains all the required attributes
-    """
-
-    clean_dict = {}
-    required_attributes = [
-        ["Name", "name"],
-        ["Description", "description"],
-        ["Condition", "itemCondition"],
-        ["Type", "@type"],
-        ["Manufacturer", "manufacturer"],
-        ["Fuel Type", "fuelType"],
-        ["Model Date", "modelDate"],
-        ["Image Url", "image"],
-        ["Vehicle Transmission", "vehicleTransmission"],
-        ["Mileage", "mileageFromOdometer"]
-    ]
-
-    for name, attribute in required_attributes:
-        value = get_attribute_safely(dictionary, attribute)
-        if value is not None:
-            clean_dict[name] = value
-
-    required_child_attributes = [
-        ["Engine Displacement", "vehicleEngine", "engineDisplacement"],
-        ["Url", "offers", "url"]
-    ]
-
-    for name, parent_attrib, child_attrib in required_child_attributes:
-        value = get_array_attribute_safely(dictionary, parent_attrib,
-                                           child_attrib)
-        if value is not None:
-            clean_dict[name] = value
-
-    price = get_array_attribute_safely(dictionary, "offers", "price")
-    currency = get_array_attribute_safely(dictionary, "offers",
-                                          "priceCurrency")
-    if price is not None and currency is not None:
-        clean_dict["Price"] = str(price) + str(currency)
-    elif price is not None:
-        clean_dict["Price"] = price
-    return clean_dict
-
-
-class PakWheelsCrawler(scrapy.Spider):
-    """This class is used to crawl website using
-    scrapy
-    """
-
-    name = "pak_wheels"
-
-    def __init__(self):
-        self.base_url = "https://www.pakwheels.com"
-        self.dots = 0
-        self.dot_limit = 5
-        super().__init__()
-
-    def start_requests(self):
-        """This method is called by spider automatically
-        to start crawl
-        """
-
-        used_cars_url = "https://www.pakwheels.com/used-cars/search/"
```

Comment:
Create a constant for  `https://www.pakwheels.com/` because it is used in multiple places